tests/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 enable_testing()
 include(GoogleTest)
 add_library(test_common_libs INTERFACE)
-target_link_libraries(test_common_libs INTERFACE pthread gtest gtest_main)
+target_link_libraries(test_common_libs INTERFACE pthread gtest gtest_main test_common_libs)
```

Comment:
`test_common_libs` depends on `test_common_libs`?

Reply:
typo, Will fix it